.env

```diff
@@ -28,5 +28,7 @@ SUBSTREAMS_API_TOKEN=<get_from_substream_site>
 EXPO_PUBLIC_IMG_PROXY_URL=https://imgproxy.tools.teritori.com/insecure/
 EXPO_PUBLIC_PUBLIC_TENOR_KEY=AIzaSyA8TpVfoyoBaDdLww6wJ1Xe0OVN-Hi8qPE
 
+#map token
+EXPO_PUBLIC_LEAFLET_MAP_TOKEN=QkwJFLzzxPan25YCgnDExGpMFPxA3x4lnyKiUf8zmaqXLP5XyOR8n3yEM8jlKV3W
```

Comment:
Is this a good idea to put the token there ? @n0izn0iz

Reply:
is it a paid api?

Reply:
Nope, we agree that is not enough sensitive to put it anywhere else 👍
We keep it like that 👍